app/src/main/res/layout/activity_Excercise_menu.xml

```diff
@@ -12,6 +12,13 @@
     android:paddingTop="@dimen/activity_vertical_margin"
     tools:context="com.edulectronics.tinycircuit.Views.excerciseMenuActivity">
 
+    <ImageView
+        android:layout_width="200dp"
+        android:layout_height="170dp"
+        app:srcCompat="@mipmap/Exercise"
+        android:layout_centerHorizontal="true"
+        android:id="@+id/imageView2" />
```

Comment:
Zou je deze naam nog aan kunnen passen zodat we ook daadwerkelijk weten waar we het over hebben als we dit id in de code gebruiken